seed/static/seed/partials/data_upload_modal.html

```diff
@@ -36,8 +38,12 @@ <h4 class="modal-title" ng-switch-when="12">Error Processing Data</h4>
             <div class="form-group col-lg-12 col-sm-12">
                 <div class="row">
                     <div class="col-lg-4" sd-uploader sourcetype="Assessed Raw" sourceprog="" sourcever="" importrecord="dataset.id" buttontext="Upload a spreadsheet" eventfunc="uploaderfunc(message, file, progress)" ng-hide="uploader.in_progress"></div>
-                    <div class="col-lg-4 orange_button" sd-uploader sourcetype="Assessed Raw" sourceprog="PortfolioManager" sourcever="1.0" importrecord="dataset.id" buttontext="Upload Portfolio Manager data" eventfunc="uploaderfunc(message, file, progress)" ng-hide="uploader.in_progress"></div>
-                    <div class="col-lg-8 green_button" sd-uploader sourcetype="Green Button Raw" sourceprog="" sourcever="" importrecord="dataset.id" buttontext="Upload Green Button Data" eventfunc="uploaderfunc(message, file, progress)" ng-hide="uploader.in_progress"></div>
+					<div class="col-lg-6 orange_button" sd-uploader sourcetype="Assessed Raw" sourceprog="PortfolioManager" sourcever="1.0" importrecord="dataset.id" buttontext="Upload Portfolio Manager data" eventfunc="uploaderfunc(message, file, progress)" ng-hide="uploader.in_progress"></div>
```

Comment:
@axelstudios can you look into this section? There are probably duplicates and the layout looks petty ugly.